lib/sandthorn/bounded_context.rb

```diff
@@ -0,0 +1,39 @@
+module Sandthorn
+  module BoundedContext
+    module ClassMethods
+      def aggregate_list
+        @aggregate_list = p_aggregate_list(self)
+      end
+
+      private
+      
+      def p_aggregate_list(bounded_context_module)
+        return [] unless bounded_context_module.respond_to?(:constants)
+        
+        classes = get_classes(bounded_context_module)
+        aggregate_list = classes.select { |item| item.include?(Sandthorn::AggregateRoot) }
+        modules = get_modules(bounded_context_module, classes)
+        
+        aggregate_list += modules.flat_map { |m| p_aggregate_list(m) }
+        
+        aggregate_list
+      end
+
+      def get_classes namespace
```

Comment:
Suggestion rename to `classes_in(namespace)`, and similar for `get_modules`. Reason: `get_classes` implies a java-style getter (i.e. gets something on itself), which this isn't.